opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/_view_instrument_match.py

```diff
@@ -60,12 +62,12 @@ def __init__(
         )
         if not isinstance(self._view._aggregation, DefaultAggregation):
             self._aggregation = self._view._aggregation._create_aggregation(
-                self._instrument, None
+                self._instrument, None, 0
```

Comment:
What does this zero mean?

Reply:
Nothing, we just need an integer there to fill the argument.